tests/Doctrine/Migrations/Tests/Tools/Console/Command/MigrationVersionTest.php

```diff
@@ -33,7 +36,11 @@ protected function setUp() : void
 
         $this->configuration = new Configuration($this->getSqliteConnection());
         $this->configuration->setMigrationsNamespace('DoctrineMigrations');
-        $this->configuration->setMigrationsDirectory(sys_get_temp_dir());
+        $migrationsDirectory = sys_get_temp_dir() . DIRECTORY_SEPARATOR . 'doctrine-migrations';
```

Comment:
good catch!

Reply:
Yup, that one gave a weird error that was hard to understand :smile: